x86/proofs/bignum_emontredc_8n.ml

```diff
@@ -1895,11 +1895,11 @@ let BIGNUM_EMONTREDC_8N_SUBROUTINE_CORRECT = time prove
                      (2 EXP (64 * val k) * val(C_RETURN s) +
                       bignum_from_memory
                         (word_add z (word(8 * val k)),val k) s)))
-           (MAYCHANGE [RIP; RSP; RDI; RSI; RAX; RDX; R8; R9; R10; R11] ,,
+           (MAYCHANGE [RSP] ,, CALLEE_SAVE_REGISTERS_UNCHANGED ,,
```

Comment:
I left `[RSP]` because technically it isn't a callee-save register. The pre-post condition seems to correctly describe the value of `RSP`.